backend/integrations/discord/admin_cog.py

```
@@ -0,0 +1,411 @@
+            return False
+
+        return True
+
+    async def cog_command_error(self, interaction: Interaction, error: Exception):
+        """Handle errors for admin commands."""
+        logger.error(f"Admin command error: {error}", exc_info=True)
+
+        if isinstance(error, app_commands.MissingPermissions):
+            error_message = "You don't have permission to use this command."
+        elif isinstance(error, app_commands.CommandInvokeError):
+            error_message = f"Command failed: {str(error.original)}"
```

Ruff (0.15.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)